src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -512,7 +512,6 @@ def update_os_patch_configuration_sub_setting(self, patch_configuration_sub_sett
             if not patch_configuration_sub_setting_found_in_file:
                 updated_patch_configuration_sub_setting += patch_configuration_sub_setting_to_update + "\n"
 
-            #ToDo: This adds some whitespace at the beginning of the first line in the settings file which is auto adjusted in the file later, so shouldn't have any issues right now. strip()/lstrip() on the string, does not work, will have to test accross versions and identify the impact
```

Comment:
Settings file does have get a whitespace because of our edits, this comment was left in as a todo for future and also as a reference point in case we see bugs because of that whitespace.

Haven't seen any bugs so far, but should we leave this in?

Reply:
We're not going to get to this with the existing backlog if there's no functional impact. A backlog item should be created if necessary.